scene/resources/2d/tile_set.h

```diff
@@ -205,7 +205,6 @@ class TileSet : public Resource {
 	Array compatibility_tilemap_map(int p_tile_id, Vector2i p_coords, bool p_flip_h, bool p_flip_v, bool p_transpose);
 #endif // DISABLE_DEPRECATED
 
-public:
```

Comment:
This one is required, previous `public` is inside `ifdef` and before it, it's `private`.